src/main/java/frc/robot/subsystems/color_wheel/ColorWheel.java

```diff
@@ -0,0 +1,90 @@
+package frc.robot.subsystems.color_wheel;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.ColorMatch;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorSensorV3;
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+
+public class ColorWheel extends SubsystemBase {
+
+    private final TalonSRX motor = new TalonSRX(Ports.ColorWheel.MOTOR);
+
+    private final I2C.Port i2cPort = I2C.Port.kOnboard;

```

Comment:
The only time you use this object is to initialize the ColorSenorV3, please move it to there, it doesn't part of the logic of the subsystem